backend/Interview.App/Services/SQLService.cs

```diff
@@ -28,4 +28,13 @@ public Task<IEnumerable<UserModel>> GetAllUsersAsync()
   {
     return Task.FromResult(_users);
   }
+
+  public void CreateUser(UserModel user)
+  {
+    SaveToTheDatabase(user);
+  }
+
+  private void SaveToTheDatabase(UserModel user)
```

Comment:
**note:** This is here to represent something that actually goes out to a database and saves the data to whatever schema we have setup